MTGAHelper.Tracker.WPF/Models/CardDraftPickWpf.cs

```diff
@@ -20,6 +20,7 @@ public class CardDraftPickWpf : CardWpf
         public string RatingSource { get; set; }
         public DraftRatingTopCard TopCommonCard { get; set; } = new DraftRatingTopCard(0, "");
 
+        public string RatingString => RatingToDisplay.Contains("N/A") ? RatingToDisplay : RatingFloat.ToString();
```

Comment:
This would need to be changed before merging as I don't want to add an additional property `RatingString`.

It seems I forgot to change the xaml to bind the TextBlock Text on the `RatingToDisplay` property.

The correct change would be to:
- Change the xaml for the TextBlock Text binding to use `RatingToDisplay` instead of `RatingFloat`
- While at it, remove the `RatingFloat` property as it's redundant and keep only the `RatingValue` property (and replace any usage of that `RatingFloat` property in the code with `RatingValue`)